PANDORA/PMHC/Model.py

```diff
@@ -10,25 +10,27 @@
 
 class Model:
 
-    def __init__(self, target, model_path='', output_dir = PANDORA.PANDORA_data, pdb=False, molpdf=0, dope=0):
-        '''__init__(self, target, model_path='', output_dir = PANDORA.PANDORA_data, pdb=False, molpdf=0, dope=0)
-         Initiate model object
-
+    def __init__(self, target, model_path='', output_dir=False, pdb=False, molpdf=0, dope=0):
+        ''' Initiate model object
         Args:
             target: Target object
-            output_dir: (string) output directory
             model_path: (string) path to hypothetical model
+            output_dir: (string) output directory
             pdb:  Bio.PDB object of the hypothetical model
             molpdf: (float) molpdf score
             dope:  (float) DOPE score
         '''
 
-
         self.target = target
         self.model_path = model_path
         self.molpdf = molpdf
         self.dope = dope
-        self.output_dir = output_dir
+
+        # Define the output directory
+        if output_dir == False:
+            self.output_dir = os.getcwd()
```

Comment:
Shouldn't the model object get as default the current working directory + the case directory?